### ISSUE TITLE
conan info does not show conditional dependencies

### ISSUE THREAD

**Issue by memsharded** [conan-io/conan#444]
For code like this.

```python
    def requirements(self):
        if self.options.fmt_external:
            self.requires("fmt/3.0.0@memsharded/testing")
```

**Comment by memsharded** [comment]
It is a small inconsistency in the input:

```bash
$ conan test_package -o spdlog:fmt_external=True
```
And info should be done without the package prefix:
```bash
$ conan info -o fmt_external=True
```
Or:
```bash
$ conan info test_package -o spdlog:fmt_external=True
```
Maybe a uniform use of options ALWAYS with the prefix should be used. What does @lasote think?

**Comment by lasote** [comment]
Yes, I remember I introduced some dirty patch to keep the possibility of not introduce the prefix before some feature that broke it. I think it's the best introduce always the prefix. less confusing. but there are too docs and help messages to review.

**Comment by memsharded** [comment]
Released in conan 0.12
